producers/http/handlers.go

```diff
@@ -182,3 +183,16 @@ func encode(v interface{}, w http.ResponseWriter) {
 		http.Error(w, "Failed to encode value to JSON", http.StatusInternalServerError)
 	}
 }
+
+func combineMessages(mm map[string]interface{}) (producers.MetricsMessage, error) {
```

Comment:
I left comments in a previous commit where this lived in a different place.  Would *really* like it if we could prevent things like this happening during a review.